.github/workflows/test-software.eessi.io.yml

```
@@ -60,8 +60,7 @@ jobs:
               echo "just run check_missing_installations.sh (should use easystacks/software.eessi.io/${{matrix.EESSI_VERSION}}/eessi-${{matrix.EESSI_VERSION}}-*.yml with latest EasyBuild release)"
               for easystack_file in $(EESSI_VERSION=${{matrix.EESSI_VERSION}} .github/workflows/scripts/only_latest_easystacks.sh); do
                   echo "check missing installations for ${easystack_file}..."
-                  ./check_missing_installations.sh ${easystack_file}
-                  ec=$?
+                  ./check_missing_installations.sh ${easystack_file} || ec=$?
```

```suggestion
                  ec=0
                  ./check_missing_installations.sh ${easystack_file} || ec=$?
```
This will default the `ec` to success